client/layout/guided-tours/tours/checklist-contact-page-tour.js

```diff
@@ -92,9 +95,11 @@ export const ChecklistContactPageTour = makeTour(
 		>
 			<Continue target="dialog-base-action-confirm" step="click-update" click>
 				{ translate(
-					'We’re all set, press {{b}}Set Featured Image{{/b}} to add this image to your page.',
+					'We’re all set, press {{setFeaturedImageButton/}} to add this image to your page.',
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'We’re all set, press {{SetFeaturedImageButton/}} to add this image to your page.' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).